src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER =",";
+    public static final String OPERATION ="buy";
+    int supplyAmount = 0;
+    int buyAmount = 0;
+    int result = 0;
+    String operation;
+    String amount;
+    List<String> fileData = null;
+    StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+
+        readDataFromTheFile(fromFile);
+
+        result = supplyAmount - buyAmount;
+
+        stringBuilder.append("supply, " + supplyAmount +
+                "buy, " + buyAmount +
+                "result, " + result);
+
+        writeToTheFile(toFile);
+    }
+
+    public void readDataFromTheFile(File fromFile) {
```

Comment:
The method 'readDataFromTheFile' should have a more informative name, like 'readTransactionData' to reflect the data it is working with.